http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -1453,6 +1453,7 @@ class LeoPubsubMessageSubscriber[F[_]](
     e match {
       case e: PubsubKubernetesError =>
         for {
+          _ <- logger.error(ctx.loggingCtx, e)("top level handler for handleKubernetesError")
```

Comment:
I think its worth explicitly logging here to make sure every app error gets logged to console, not just saved to db